.github/workflows/lint-and-format.yaml

```diff
@@ -25,6 +25,19 @@ jobs:
           node-version: 'lts/*'
           cache: 'npm'
 
+      - name: Cache tool outputs
+        uses: actions/cache@v4
+        with:
+          path: |
+            node_modules/.cache
+            .eslintcache
+            .cache
+            tsconfig.tsbuildinfo
+          key: ci-tools-cache-${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('src/**/*.ts', 'src/**/*.vue', '*.config.*') }}
```

Comment:
Just an observation: Having to enumerate the file types feels a little brittle :-/